cpp/harvest/core.cc

```diff
@@ -313,28 +313,126 @@ void Filter(int argc, char **argv) {
             continue;
         }
 
-        const std::string control_number(ConvertId(std::to_string(work.getId())));
-        if (ignore_unique_id_dups and unique_id_to_date_map.keyIsPresent(control_number)) {
-            work.setFilteredReason("Duplicate");
-            CORE::OutputFileAppend(filter_file, work, skipped == 0);
-            ++skipped;
-            ++skipped_dupe_count;
-            continue;
-        }
+        // const std::string control_number(ConvertId(std::to_string(work.getId())));
+        // if (ignore_unique_id_dups and unique_id_to_date_map.keyIsPresent(control_number)) {
+        //     work.setFilteredReason("Duplicate");
+        //     CORE::OutputFileAppend(filter_file, work, skipped == 0);
+        //     ++skipped;
+        //     ++skipped_dupe_count;
+        //     continue;
+        // }
+        ++record_count;
         CORE::OutputFileAppend(output_file, work, first);
         first = false;
     }
     CORE::OutputFileEnd(output_file);
     CORE::OutputFileEnd(filter_file);
 
     LOG_INFO(
+        "Number or record: " + std::to_string(record_count) + " records.\n"
+        "Filtered " + std::to_string(skipped) + " records, thereof:\n"
+        "- " + std::to_string(skipped_uni_tue_count) + " Uni Tübingen\n"
+        "- " + std::to_string(skipped_incomplete_count) + " incomplete\n"
+        "- " + std::to_string(skipped_dupe_count) + " duplicate\n"
+        "- " + std::to_string(skipped_language_count) + " language"
+    );
+}
+void FilterExecWithFilterOption(char **argv) {
```

Comment:
Most important, as discussed => the 2 split Filter functions should be merged back into 1.